source/retryable-reads/retryable-reads.rst

```diff
@@ -163,6 +163,7 @@ Drivers SHOULD support retryability for the following operations:
 - Any driver that provides generic command runners for read commands (with logic
   to inherit a client-level read concerns) SHOULD implement retryability for the
   read-only command runner.
+- Any error that occurs during a handshake (eg. Auth, ShutdownInProgress).
```

Comment:
I'm not sure this is the right section of the spec for this prose. This section says "Drivers SHOULD support retryability for the following operations" whereas this spec change is defining a new type of retryable error that MUST be retried. What if we add this to the "Retryable Error" definition:
```
...
- a `PoolClearedError`_
- Any of the above retryable errors that occur during a connection handshake (including the
  authentication step). For example, a network error or ShutdownInProgress error
  encountered when running the hello or saslContinue commands.
```

Reply:
This was still left over from when I started this spec change as a should/may 😅 good catch, moved!